index.d.ts

```diff
@@ -0,0 +1,3 @@
+export { extractCss } from './core/update';
+export { styled, setup } from './styled';
+export { css, glob, keyframes } from './css';
```

Comment:
Oh this is generated by microbundle but I've added a flag to not get this anymore. Can you check that you have this line in your fork? https://github.com/cristianbote/goober/blob/master/package.json#L75

Reply:
Yes, I have the same package.json, but this file is still being created